### PR TITLE
docs: Updated README `getting started` section

### PR DESCRIPTION
**Pull request by yukinoda**
`yarn` is commonly used as well in react projects, it'd be good to have the installation guide using it

Added `yarn` installation guide and link to the npm package

related issue: #8291

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @yukinoda!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/docs/readme-getting-started/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Review by yukinoda — COMMENTED** [review]
Ready to be reviewed

**Comment by yukinoda** [comment]
this is ready

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -170,6 +170,7 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to contribute to Cesiu
 
 ## [Individual CLA](Documentation/Contributors/CLAs/individual-contributor-license-agreement-v1.0.pdf)
 
+- [Yuki Noda](https://github.com/yukinoda)
```
Comment (ebogo1) [inline CONTRIBUTORS.md:173]: Please move your entry to the bottom of the contributor's list.

File: README.md [hunk 2]
```diff
@@ -14,10 +14,14 @@ CesiumJS is a JavaScript library for creating 3D globes and 2D maps in a web bro
 
 ### :rocket: Get Started
 
-Visit the [Downloads page](https://cesium.com/downloads/) or use the npm module:
+Visit the [Downloads page](https://cesium.com/downloads/) or install the [npm package](https://www.npmjs.com/package/cesium):
```
Comment (ebogo1) [inline README.md:17]: I am a fan of the link to the cesium package on the npm registry, but I don't think we want the instructions for `yarn` in our README.md. We'd like to keep the instructions as simple as possible so the extra lines for yarn don't seem like they fit. 

The "```sh" is a nice addition though :+1:

**Review by ebogo1 — COMMENTED** [review]
Thanks @yukinoda! Some comments, otherwise this is good to go:

File: CONTRIBUTORS.md (hunk 1 above)
Comment (yukinoda) [reply CONTRIBUTORS.md]: 21887a5

File: README.md (hunk 2 above)
Comment (yukinoda) [reply README.md:17]: 09b3826

**Review by yukinoda — COMMENTED** [review]
@ebogo1 
thanks for the comments, I've reflected them.

**Pull request merged**